src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.springboot.datajpa.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "category")
```

Comment:
Table namings are inconsistent. For example `category` is in a singular form, but `products` is in plural form. Let's choose one way